yarn.lock

```diff
@@ -3464,6 +3469,305 @@
     prop-types "^15.6.1"
     react-lifecycles-compat "^3.0.4"
 
+"@react-aria/checkbox@3.0.0-rc.4":
```

Comment:
yarn lock seems wrong again, it shouldn't be pulling in our packages from the public registry